### PR TITLE
Update caching.rst

### PR DESCRIPTION
**Pull request by ReDnAxE**
Updating doc for this PR Making cache PSR6 compliant : [#19741](https://github.com/symfony/symfony/pull/19741)

### CODE REVIEW

File: components/expression_language/caching.rst [hunk 1]
```diff
@@ -39,12 +37,6 @@ in the object using the constructor::
     $cache = new MyDatabaseParserCache(...);
     $language = new ExpressionLanguage($cache);
```
Comment (wouterj) [inline components/expression_language/caching.rst]: Can we maybe enhance this a bit, to show we're really doing PSR-6 here?

E.g. by showing something like:
```php
use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
use Symfony\Component\Cache\RedisAdapter;

$cache = new RedisAdapter(...);
$language = new ExpressionLanguage($cache);
```

Then, we can add a small "seealso" box pointing to the cache component:

```rst
.. seealso::

    See the :doc:`/components/cache` documentation for more information about
    available cache adapters.
```

**Comment by wouterj** [comment]
The feature is now merged, congratz @ReDnAxE!

Thanks a lot for submitting a doc PR alongside your code PR. I've suggest some changes. If you have some time, please apply the suggestions if you agree with them. Otherwise, you can just comment and we'll finish this PR so it's merged before the 3.2 stable release.

status: needs work

File: components/expression_language/caching.rst (hunk 1 above)
Comment (ReDnAxE) [inline components/expression_language/caching.rst]: Thank you for your help, I agree, I'll do it as soon as possible :)

**Comment by ReDnAxE** [comment]
I just applied your recommendations ^^, you think it is enough ?

status: needs review

**Comment by wouterj** [comment]
👍 
status: reviewed

I think it's perfect now. Thanks!

**Comment by javiereguiluz** [comment]
👍

**Comment by wouterj** [comment]
Thanks for documenting the feature you submitted! I've merged this into the docs now and made some very minor changes in 31939d5 Hope to see you back in the future!

**Pull request merged**